### ISSUE TITLE
Recommend geography-based channels to new members

### ISSUE THREAD

**Issue by hollomancer** [OperationCode/operationcode_pyback#60]
When a new user is greeted, given that we know their zip code, we can recommend #city/#state Slack channels to them via a DM from the bot - without needing a human to make the recommendation.

I was actually wondering if we could make a user join a channel via the bot, but I am not sure if the API allows that.

We can just start with our biggest meetups now, as an MVP - NYC and Seattle.

* NYC: https://www.health.ny.gov/statistics/cancer/registry/appendix/neighborhoods.htm
* Seattle: `98101, 98102, 98103, 98104, 98105, 98106, 98107, 98108, 98109, 98112, 98115, 98116, 98117, 98118, 98119, 98121, 98122, 98125, 98126, 98133, 98134, 98136, 98144, 98146, 98154, 98164, 98174, 98177, 98178, 98195, 98199`

**Comment by AllenAnthes** [comment]
@hollomancer  Looks like we can use [channels.invite](https://api.slack.com/methods/channels.invite) .  If I remember correctly "inviting" essentially joins the channel for the user, no?

**Comment by hollomancer** [comment]
Yep, pretty much. That seems like a good solution.

**Comment by wimo7083** [comment]
Blocked by backend integration. Need ability to get user information when they are registered. Alternative way is using the admin status to get the details (ip address and relate that to the locations). That is a route that I'm hesitant to do.

**Comment by hollomancer** [comment]
Is there already a backend issue for that?

**Comment by wimo7083** [comment]
Yes, there's a few that are working on integration. The first step was to get the simple interface setup with the tables to give parity to how BE/ bot store emails and use that as a key. Once that was done a better connection was going to be looked at.
